samza-core/src/main/java/org/apache/samza/standalone/PassthroughJobCoordinator.java

```diff
@@ -154,4 +155,9 @@ private String createProcessorId(Config config) {
               ApplicationConfig.APP_PROCESSOR_ID_GENERATOR_CLASS));
```

Comment:
Same here, use method instead of directly referring to the config name.

Reply:
Sure. These are not my changes. But refactored code a bit here.